raster-foundry/raster-foundry
# Migration to split Sentinel-2A and Sentinel-2B datasources

## Overview

This PR contains a migration for creating a separate 'Sentinel-2B' datasource and renaming the original datasource to 'Sentinel-2A'.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~


## Testing Instructions

 * Run migrations and see that there are now 2 datasources for sentinel

Closes #2785
